cv/lane_detection/src/lane_detection_inferencer.cpp

```diff
@@ -0,0 +1,242 @@
+#include "lane_detection/lane_detection_inferencer.h"
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <functional>
+#include <numeric>
+#include <stdexcept>
+#include <vector>
+
+#include "onnxruntime_c_api.h"
+#include "onnxruntime_cxx_api.h"
+#include "opencv2/opencv.hpp"
+
+namespace
+{
+
+constexpr int NETWORK_INPUT_WIDTH{330};
+constexpr int NETWORK_INPUT_HEIGHT{180};
+constexpr int NETWORK_INPUT_CHANNEL{5};
+constexpr int NETWORK_OUTPUT_WIDTH{330};
```

Comment:
We probably shouldn't have a separate constant for input and output in hindsight (since we're doing segmentation here)